### ISSUE TITLE
Debug mode logs

### ISSUE THREAD

**Issue by shanemeyers** [ItzAmirreza/EzChestShop#45]
I'm seeing an error in my logs, but I can't find a debug mode to enable to try to find what's wrong with the shop. Please add a debug log level to the config so I can get more detail about what is going on surrounding the error message. I'm getting many of:

```
[22:20:47] [Server thread/INFO]: [ECS] Something unexpected happened with this container's data, so this shop has been removed.
[22:21:09] [Server thread/INFO]: [ECS] Something unexpected happened with this container's data, so this shop has been removed.
```

EzChestShop - version 1.5.0

**Comment by ItzAmirreza** [comment]
Received 1 more report regarding this. Investigating...

**Comment by ItzAmirreza** [comment]
Fixed in 1.5.3 version.
https://www.spigotmc.org/resources/ez-chest-shop-ecs-1-14-x-1-19-x-mega-update.90411/
https://modrinth.com/plugin/ezchestshop
https://www.curseforge.com/minecraft/bukkit-plugins/ez-chest-shop-1-14-x-1-19-3
